docs/admin/overview.md

```diff
@@ -0,0 +1,22 @@
+---
+id: overview
+title: "Administration"
+sidebar_label: "Administration"
+---
+
+The admin section of Scoreboard allows admin users to manage badges, teams, and users.
+
+## [Badges](./badges)
```

Comment:
These individual overview pages do not exist for me; for example, `docs/admin/badges` gives me a "Not Found" error